docs/development/README.adoc

```diff
@@ -1,9 +1,12 @@
-:toc: macro
+:toc: left
+:toclevels: 3
+:sectanchors: true
+:sectids: true
+:source-highlighter: rouge
+:icons: font
 
 = T developer documentation
 
-toc::[]
-
 == <<local-t-network.adoc,Setting up local T network>>
 
 For development purposes, it is convenient to set up a local Ethereum client with
```

Comment:
Also, a couple of notes for the `Quick installation` section (I couldn't comment on specific lines):
* We say "To quickly install and **start** a client use the following scripts". I don't think we have the part about starting the client covered below.
* I would move the part starting with `To deploy and initialize contracts:` to the beginning of `Install scripts` section. If not, I would provide the full path for the install files ( i.e. `scripts/install.sh` and `scripts/initialize.sh`) mentioned in the first parts of the section.

Reply:
The development docs are out of the scope of this PR. Can we handle the updates in a separate PR?